common/components/stores/ProjectSearchStore.js

```diff
@@ -80,17 +83,13 @@ class ProjectSearchStore extends ReduceStore<State> {
     }
   }
 
-  _clearProjects(state: State): State {
-    return state.set('projects', null);
-  }
-
-  _loadProjects(state: State): void {
-    const {keyword, issueArea} = state;
+  _loadProjects(state: State): State {
     const url = [
       '/api/projects?',
-      keyword ? '&keyword=' + keyword : null,
-      issueArea
-        ? '&issueArea=' + lodash.snakeCase(issueArea)
+      state.keyword ? '&keyword=' + this.getState().keyword : null,
+      state.tags
+        ? '&tags='
+          + state.tags.map(tag => tag.tagName).join(',').replace('-', '_')
```

Comment:
Why replace the tag name dashes with underscores?  We use dashes in tag names on the backend.

Reply:
Previous to 55d2c1a29ce68c02a883d4303b1878d1b2cea4f7, tags used underscores. Will update UI to reflect backend change.

Reply:
resolved by 878e7756fca87431eadc179a31342c85b4faa89f